### PR TITLE
D3ASIM-3218: Omitting reading general settings in the redis case.

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

**Review by Hassan754 — APPROVED** [review]
I was in the same spot just before the standup, Thanks for fixing it !

**Review by fievelk — APPROVED** [review]

File: d3a_api_client/cli.py [hunk 1]
```diff
@@ -91,21 +91,22 @@ def run(base_setup_path, setup_module_name, username, password, domain_name, web
...
+            os.environ["API_CLIENT_SIMULATION_ID"] = config["uuid"]
+        else:
+            os.environ["API_CLIENT_DOMAIN_NAME"] = domain_name \
+                if domain_name else domain_name_from_env()
+            os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"] = web_socket \
+                if web_socket else websocket_domain_name_from_env()
+            os.environ["API_CLIENT_SIMULATION_ID"] = simulation_id \
+                if simulation_id else simulation_id_from_env()
+
+        validate_settings_are_set_before_launch()
```
Comment (Hassan754) [inline d3a_api_client/cli.py:109]: This should be unindented

Comment (hannesdiedrich) [reply d3a_api_client/cli.py:109]: @Hassan754  not really, have a loot into the function. All the settings are not needed for the redis case.

Comment (hannesdiedrich) [reply d3a_api_client/cli.py:109]: I should remove `API_CLIENT_RUN_ON_REDIS` from `settings_list` though

Comment (hannesdiedrich) [reply d3a_api_client/cli.py]: https://github.com/gridsingularity/d3a-api-client/pull/129/commits/793dac45dd85889b2692d0eaa77c0688d8b9e112

**Review by Hassan754 — APPROVED** [review]

**Review by fievelk — APPROVED** [review]

**Pull request merged**